src/main/webapp/projectContainer/ConsentGroups.js

```diff
@@ -61,6 +62,69 @@ export const ConsentGroups = hh(class ConsentGroups extends Component {
       }).parent().removeClass("ui-widget-content");
       $(".ui-dialog-titlebar").hide();
     });
+
+    $(".confirmationModal").on('click', function () {
+      $("#confirmation-modal-dialog").load(
+        component.serverURL + "/api/consent-group/confirmation-modal?"
```

Comment:
Ah ... I think I see why you added the gsp. I don't agree with this approach ... all new API calls should be returning raw json objects, not gsps. All html view content should be a react component.